server/controllers/messagesController.js

```diff
@@ -10,9 +10,9 @@ const db = require('../models');
 const {Op} = require("sequelize");
 const Message = db.message;
 
-const createMessage = (req,uid) => {
```

Comment:
Nice fix there!

Reply:
Thanks - this is a result of us having access to our own uid through the promise.